packages/office-addin-lint/src/lint.ts

```diff
@@ -34,6 +34,7 @@ export function performLintCheck(files: string) {
   } catch (err) {
     if (err.status && err.status == ESLintExitCode.HasLintError) {
       usageDataObject.sendUsageDataSuccessEvent("performLintCheck", { exitCode: ESLintExitCode.HasLintError });
+      throw err;
```

Comment:
This makes sense. If you notice now, both the if and the else throw err. You can simplify this to always throw error after the if/else. The point here is that the exception is caught in order to send the usage data, but needs to be rethrown so the error is displayed by the top-level catch for the command.

Reply:
(This applies for of the places.)